woocommerce-stats-by-currency.php

```diff
@@ -124,6 +124,70 @@ function wsbc_add_stats() {
+                                }
+                                .button {
+                                    display: inline-block;
+                                    text-decoration: none;
+                                    font-size: 13px;
+                                    line-height: 2.15384615;
+                                    min-height: 30px;
+                                    margin: 0;
+                                    padding: 0 10px;
+                                    cursor: pointer;
+                                    border-width: 1px;
+                                    border-style: solid;
+                                    -webkit-appearance: none;
+                                    border-radius: 3px;
+                                    white-space: nowrap;
+                                    box-sizing: border-box;
+                                }
+                                .primary {
+                                    background: #007cba;
+                                    border-color: #007cba;
+                                    color: #fff;
+                                    text-decoration: none;
+                                    text-shadow: none;
+                                }
+                                .primary:hover {
+                                    background: #0071a1;
+                                    border-color: #0071a1;
+                                    color: #fff;
+                                }
+                            </style>';
+        $warning_logo   = '<svg xmlns="http://www.w3.org/2000/svg" width="54px" height="54px" viewBox="0 0 64 64"><defs><linearGradient gradientTransform="matrix(1.31117 0 0 1.30239 737.39 159.91)" gradientUnits="userSpaceOnUse" id="0" y2="-.599" x2="0" y1="45.47"><stop stop-color="#ffc515"/><stop offset="1" stop-color="#ffd55b"/></linearGradient></defs><g transform="matrix(.85714 0 0 .85714-627.02-130.8)"><path d="m797.94 212.01l-25.607-48c-.736-1.333-2.068-2.074-3.551-2.074-1.483 0-2.822.889-3.569 2.222l-25.417 48c-.598 1.185-.605 2.815.132 4 .737 1.185 1.921 1.778 3.404 1.778h51.02c1.483 0 2.821-.741 3.42-1.926.747-1.185.753-2.667.165-4" fill="url(#0)"/><path d="m-26.309 18.07c-1.18 0-2.135.968-2.135 2.129v12.82c0 1.176.948 2.129 2.135 2.129 1.183 0 2.135-.968 2.135-2.129v-12.82c0-1.176-.946-2.129-2.135-2.129zm0 21.348c-1.18 0-2.135.954-2.135 2.135 0 1.18.954 2.135 2.135 2.135 1.181 0 2.135-.954 2.135-2.135 0-1.18-.952-2.135-2.135-2.135z" transform="matrix(1.05196 0 0 1.05196 796.53 161.87)" fill="#000" stroke="#40330d" fill-opacity=".75"/></g></svg>';
+        $msg_text       = '<p>Sorry, This plugin requires <strong>WooCommerce</strong> and <strong>WooCommerce Price Based on Country</strong> plugins to work. Learn how to install <strong>WooCommerce Stats By Currency</strong> in the <a href="https://lubus.in/blog/woocommerce-stats-by-currency#guide" target="_blank">guide</a>.</p>';
```

Comment:
Currently, the plugin doesn't require setup guide and we don't have one. so let's remove `Learn how to install WooCommerce Stats By Currency in the guide.`